sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md

```diff
@@ -94,10 +144,207 @@ directive:
         $["x-ms-client-default"] = 1;
 ```
 
+### Fix `enum` error
+
 ```yaml
 directive:
   - from: swagger-document
     where: $["definitions"]["CustomConversationTaskParameters"]
     transform: >
         delete $.properties["stringIndexType"]
```

Comment:
I think in QnA we had to compensate for deleting this property by patching the client to set it explicitly. Could we copy that logic from the questionanswering SDK to apply it here?
Here's the patch:
https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/cognitivelanguage/azure-ai-language-questionanswering/azure/ai/language/questionanswering/models/_patch.py#L68-L70

Reply:
good catch!

Reply:
not valid as we're not generating enums anymore